app/src/main/res/layout/item_word_list_home.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+
+    >
+    <data>
+        <variable
+            name="item"
+            type="com.vinh.dictionary_1.data.model.Word"
+            />
+
+        <variable
+            name="itemListener"
+            type="com.vinh.dictionary_1.screen.BaseRecyclerViewAdapter.OnRecyclerViewItemClickListener"
+            />
+    </data>
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/dp_15"
+        android:layout_marginLeft="@dimen/dp_15"
+        android:layout_marginRight="15dp"
+        android:layout_marginStart="@dimen/dp_15"
+        android:background="@color/color_light_grey"
+        android:orientation="vertical"
+        android:layout_marginTop="@dimen/dp_10"
+        android:onClick="@{(v) -> itemListener.onItemRecyclerViewClick(v, item)}"
+        >
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            >
+
+            <TextView
+                android:id="@+id/text_word"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:maxLength="20"
+                android:maxLines="1"
+                android:textColor="@color/color_black"
+                android:textSize="@dimen/sp_17"
+                android:textStyle="bold"
+                android:text="@{item.word}"
+                />
+
+            <TextView
```

Comment:
WARNING: [RelativeOverlap] `@id/text_pronounce` can overlap `@id/text_word` if @{item.word}, @{item.pronounce} grow due to localized text expansion
 If relative layout has text or button items aligned to left and right sides they can overlap each other due to localized text expansion unless they have mutual constraints like `toEndOf`/`toStartOf`.